exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -118,19 +120,25 @@ func newPRWExporter(cfg *Config, set exporter.Settings) (*prwExporter, error) {
 		return nil, err
 	}
 
+	err = config.RemoteWriteProtoMsg.Validate(cfg.RemoteWriteProtoMsg)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Bit more readable + scoped error (it's clear err value is not used somewhere else).

```suggestion
	if err := config.RemoteWriteProtoMsg.Validate(cfg.RemoteWriteProtoMsg); err != nil {
		return nil, err
	}
```